KD_Lib/original/mnist.py

```diff
@@ -0,0 +1,73 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import datasets, transforms
+
+from train import train_teacher, train_student
+from evaluate import eval
+from model import teacher, student
+
+import argparse
+
+
+def mnist():
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+
+    parser = argparse.ArgumentParser(description='MNIST implementation of the "Distilling the Knowledge in a Neural Network Paper"')
+
+    parser.add_argument('--epochs', default=20, 
+                        help='Default:20')
+    parser.add_argument('--batch_size', default=100, 
+                        help='Default:100')
+    parser.add_argument('--lr', default=0.01,
+                        help='Default:0.01')
+    parser.add_argument('--loss_fn', default='MSE', 
+                        help='(Options:MSE,KL), (Default:MSE)')
+    parser.add_argument('--optim', default='SGD',
+                        help='(Options:SGD,Adam), (Default:SGD)')
+    parser.add_argument('--teacher_size', default=1200, 
+                        help='Default:1200')
+    parser.add_argument('--student_size', default=800, 
+                        help='Default:800')
+    parser.add_argument('--distill_weight', default=0.7, 
+                        help='Default:0.7')
+    parser.add_argument('--temp', default=20, 
+                        help='Default:20')
+    args = parser.parse_args()
```

Comment:
Can you convert these to function arguments? We won't be primarily using these functions from command line.